examples/flutter_gallery/lib/demo/material/elevation_demo.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+
+class ElevationDemo extends StatefulWidget {
+  static const String routeName = '/material/elevation';
+
+  @override
+  State<StatefulWidget> createState() => _ElevationDemoState();
+}
+
+class _ElevationDemoState extends State<ElevationDemo> {
+  bool _shouldShowElevation = true;
```

Comment:
This could be just `_showElevation`, since it's not a recommendation :-).

Reply:
Done.